src/hooks/useUserGauge.ts

```diff
@@ -54,6 +57,18 @@ export default function useUserGauge(gaugeAddress?: string): UserGauge | null {
     userGaugeRewards?.claimableExternalRewards.length,
   )
 
+  const boost =
+    gauge.gaugeBalance?.gt(Zero) && gauge.workingBalances?.gt(Zero)
+      ? `${formatUnits(
+          gauge.workingBalances
+            .mul(BN_1E18)
+            .div(gauge.gaugeBalance)
```

Comment:
Careful when dividing by unknown values - they may be 0 and BigNumber will throw

Reply:
we have a null and non Zero check above.